quickjs-libc.c

```diff
@@ -940,6 +947,7 @@ static JSValue js_std_popen(JSContext *ctx, JSValue this_val,
     JS_FreeCString(ctx, mode);
     return JS_EXCEPTION;
 }
+#endif
```

Comment:
```suggestion
#endif // !defined(__wasi__)
```
Some of those blocks are pretty big and it's not always obvious what if statement an endif belongs to.